modules/tinymce/CHANGELOG.md

```diff
@@ -29,7 +29,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The context toolbar will now move out of the way when it overlaps with the selection, such as in table cells #TINY-7192
 
 ### Changed
-- Changed the load order so that the content css gets loaded before the editor gets populated with contents. #TINY-7249
+- Changed the load order so that the content css gets loaded before the editor gets populated with contents #TINY-7249
+- Changed emoticons, wordcount, code, and codesample plugins to open dialogs using commands #TINY-7619
```

Comment:
This also needs to mention that we have the 2 new commands: `mceEmoticons` and `mceWordCount`. However, that should be under the `Added` section instead, so we'll need 2 changelog entries here.

- [ ] Add a changelog entry for the new commands